ui-particles-angular/projects/ui-particles-angular/src/lib/gio-menu/gio-menu.component.scss

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@use 'sass:map';
+@use '@angular/material' as mat;
+
+@use '../../scss' as gio;
+
+$background: map.get(gio.$mat-space-palette, lighter10);
+
+.gio-menu {
+  display: flex;
+  width: 224px;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 16px 0;
+  background: $background;
+  box-shadow: 0 0 0 1px rgb(0 0 0 / 10%), 0 5px 10px rgb(0 0 0 / 20%), 0 15px 40px rgb(0 0 0 / 40%);
```

Comment:
it would be possible to replace the colors by the one of Per" ? maybe with alpha (last param of `get-color-from-palette`)